integration-tests/testconfig/default.toml

```diff
@@ -1,3 +1,8 @@
+# This is the default configuration so OCR2 tests can run without issues
+[ChainlinkImage]
+image="public.ecr.aws/chainlink/chainlink"
+version="2.9.0"
```

Comment:
this gets overridden by the CI right? this is mostly if someone is interested in running a test locally?

Reply:
Yes, this is the default config so you don't have to set anything up to run locally, by default it will kick off tests on localnet